content/guides/embed/ui-elements/preview.md

```diff
@@ -317,6 +317,89 @@ preview.addListener("rotate", data => {
 });
 ```
 
+## Annotations
+
+You can enable V4 [annotations][annotationsguide] in content preview. New
+annotations will sync in real time.
+
+To add V4 annotations to preview:
+
+1. Run `npm i box-annotations@latest`to install [box annotations][annotations].
+<Message warning>
+Box annotations version should be at least major version 4 and up.
+</Message>
+2. Run `npm i box-ui-elements@15.0.0` to install [BUIE][buie] version with
```

Comment:
```suggestion
2. Run `npm i box-ui-elements@16.0.0` to install [BUIE][buie] version with
```

We are going to release BUIE 16 to include these changes, 15 is the current major release

Reply:
https://jira.inside-box.net/browse/DDOC-688